src/renderers/shared/server/ReactDOMNodeStreamRenderer.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright 2013-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule ReactDOMNodeStreamRenderer
```

Comment:
This file should live in the DOM folder. The only reason the other one wasn't is that we expect that most of the logic can be extracted as separate renderer and then move some parts out to DOM.